contrib/jss/invoke.lisp

```diff
@@ -205,15 +227,19 @@ NAME can either string or a symbol according to the usual JSS conventions."
 ;; a regular Lisp string as ABCL converts the Java string to a Lisp string.
 
 (eval-when (:compile-toplevel :load-toplevel :execute)
-  (defun read-invoke (stream char arg) 
-    (unread-char char stream)
-    (let ((name (read stream)))
-      (if (or (find #\. name) (find #\{ name))
-          (jss-transform-to-field name arg)
-          (let ((object-var (gensym))
-                (args-var (gensym)))
-            `(lambda (,object-var &rest ,args-var) 
-               (invoke-restargs ,name  ,object-var ,args-var ,(eql arg 0)))))))
+  (defun read-invoke (stream char arg)
+    (if (eql arg 1)
+        (progn (require 'javaparser)
```

Comment:
1.  `javaparser` is an ASDF system so please use `asdf:load-system`

2.  Thunking through the load routines each time `read-invoke` is gonna be expensive isn't it?  Maybe just move the `javaparser` into a dependent system of `jss` as `jss/javaparser` always loading this?